packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -37,20 +36,33 @@ interface ComboboxBaseProps<T extends string | number>
    */
   createCustomValue?: boolean;
   options: Array<ComboboxOption<T>> | ((inputValue: string) => Promise<Array<ComboboxOption<T>>>);
-  onChange: (option: ComboboxOption<T> | null) => void;
+  /**
+   * The onChange handler is called with `null` when clearing the Combobox.
+   */
+  onChange: (option: ComboboxOption<T>) => void;
   /**
    * Most consumers should pass value in as a scalar string | number. However, sometimes with Async because we don't
    * have the full options loaded to match the value to, consumers may also pass in an Option with a label to display.
    */
-  value: T | ComboboxOption<T> | null;
+  value?: T | ComboboxOption<T> | null;
   /**
    * Defaults to 100%. Number is a multiple of 8px. 'auto' will size the input to the content.
    * */
   width?: number | 'auto';
+  placeholder?: InputProps['placeholder'];
```

Comment:
Any thoughts on inferring these from `Input`? We're passing them on regardless, and they are all strings, so we could just put them as `string`.

Reply:
Why not just use `Pick` from InputProps?